VERSION

```diff
@@ -1 +1 @@
-2.9.11
+2.9.12
```

Comment:
This should only be done in a release PR, as we don't have a changelog for `2.9.12` yet (see [docs/changelog.rst](https://github.com/juju/python-libjuju/blob/2.9/docs/changelog.rst)).